apps/demos/Demos/DataGrid/CustomEditors/jQuery/index.js

```diff
@@ -113,7 +114,7 @@ $(() => {
           scrolling: { mode: 'virtual' },
           height: 250,
           selection: { mode: 'single' },
-          selectedRowKeys: [cellInfo.value],
+          selectedRowKeys: selectedRowKeys,
```

Comment:
Minor comment.
(To fix or not is at the discretion of the PR author)

We can write the same object property assignation in a shorter way:
```suggestion
          selectedRowKeys,
```